pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go

```diff
@@ -145,12 +149,17 @@ func (a *dataFeedsAggregator) Aggregate(previousOutcome *types.AggregationOutcom
 	toWrap := []any{}
 	for _, report := range reportsNeedingUpdate {
 		feedID := datastreams.FeedID(report.FeedID).Bytes()
+		remappedID := a.config.Feeds[datastreams.FeedID(report.FeedID)].RemappedID
```

Comment:
this is odd. why not use the canonical 
x, exists := mymap[y]
if !exists { x = ...}

Reply:
This is not a check for existence in a map - it has to exist at this point. RemappedID field can simply be empty and then we fall back to original ID (I'll add a comment)